maybe/syscall_filters.py

```diff
@@ -95,6 +96,17 @@ def format_open(path, flags):
         return None
 
 
+def format_mknod(path, type):
+    path = abspath(path)
+    if (type & S_IFREG):
+        return "%s %s" % (T.red("create file"), T.underline(path))
```

Comment:
The color should be cyan just as it is for the existing create file/directory/link output.